src/lib.rs

```diff
@@ -13,6 +13,74 @@ mod util;
 
 pg_module_magic!();
 
+/// TODO: presently, we deliberate on how to structure SQL-only tests.
+/// As soon as we agree on something these tests should be moved over.
+
+#[cfg(any(test, feature = "pg_test"))]
+#[pg_schema]
+mod tests {
+    use pgx::*;
+
```

Comment:
can we add a test using values of all the supported types to make sure the hash works for each

1. string
2. int
3. float
4. bool
5. etc...

Reply:
Could you have a look at the tests I've implemented for the `jsonb_digest` itself and see if they suffice or you'd like me to add something? #149 includes a bunch of hand-coded cases as well as a property-based (randomized) test that checks 1000 cases on each run.

https://github.com/timescale/promscale_extension/pull/149/files#diff-07dec81586875043695312db239d17a9ea2ee59473fb436fe0234e4a4339df2dR72

Reply:
lgtm